### PR TITLE
Update interbit-ui-components Contributing doc

### PR DESCRIPTION
**Pull request by karikushi**
The Contributing doc is updated with information about how UI components should manage state.

### CODE REVIEW

File: reference/interbit-ui-components/contributing.adoc [hunk 1]
```diff
@@ -79,6 +79,14 @@ Simple, presentational components should go in the `src/components/UIKit`
 directory. More complex components should be in their own directory within
 `src/components`. Each component must be exported from `src/index.js`.
 
+Presentational components should be dumb components where possible. They:
```
Comment (nicholeous) [inline reference/interbit-ui-components/contributing.adoc]: components ~where possible~.

File: reference/interbit-ui-components/contributing.adoc [hunk 2]
```diff
@@ -79,6 +79,14 @@ Simple, presentational components should go in the `src/components/UIKit`
...
+
+- Receive data and callbacks as props exclusively.
+- Don't have their own state.
+- If components must have their own UI-related state (e.g. whether a modal is
```
Comment (nicholeous) [inline reference/interbit-ui-components/contributing.adoc]: +1 Listing requirements. :smile_cat:

**Review by nicholeous — CHANGES_REQUESTED** [review]

**Review by nicholeous — APPROVED** [review]
:smile_cat:

File: reference/interbit-ui-components/contributing.adoc (hunk 2 above)
Comment (btlewan) [reply reference/interbit-ui-components/contributing.adoc]: Nit: Each point is part of a sentence that begins with 'They', and so should not have the first word capitalized.

**Review by btlewan — APPROVED** [review]
One really minor suggestion. No problem if you ignore it. :-)

File: reference/interbit-ui-components/contributing.adoc (hunk 2 above)
Comment (karikushi) [reply reference/interbit-ui-components/contributing.adoc]: It's corrected now 😄

**Pull request merged**